src/App.tsx

```diff
@@ -17,18 +33,32 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter setTodos={setTodos} />
```

Comment:
The TodoFilter component appears to be directly manipulating the `todos` state of the parent App component. This is not a recommended practice in React. Instead, you should pass down a handler function that encapsulates any state logic.